tron/bin/check_tron_datastore_staleness.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3.6
+import argparse
+import logging
+import os
+import sys
+import time
+
+import pytz
+
+from tron.config import manager
+from tron.config import schema
+from tron.serialize.runstate.dynamodb_state_store import DynamoDBStateStore
+
+# Default values for arguments
+DEFAULT_WORKING_DIR = '/var/lib/tron/'
+DEFAULT_CONF_PATH = 'config/'
+DEFAULT_STALENESS_THRESHOLD = 30
+log = logging.getLogger('tron_stateless_alert')
```

Comment:
Why not use `__name__`?
Otherwise you should update this now, since the name of the script has changed.

Reply:
Shouldn't __name__ be '__main'__'? Thx for pointing it out, I shall update it.